libs/application/templates/new-primary-school/src/lib/messages.ts

```diff
@@ -461,6 +461,18 @@ export const newPrimarySchoolMessages: MessageDir = {
       defaultMessage: 'Skólaakstur',
       description: 'School bus',
     },
+    schoolBusTitle: {
+      id: 'dess.nps.application:different.needs.school.bus.title',
+      defaultMessage: 'Upplýsingar um skólaakstur',
+      description: 'Information about school bus',
+    },
+    schoolAlertMessage: {
```

Comment:
Maybe change the name?
```suggestion
    schoolBusAlertMessage: {
```

Reply:
Fixed